tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3784,20 +3815,52 @@ <h2>Show similarity to selected datapoint</h2>
           }
         },
 
+        isSameInferenceClass_: function(val1, val2) {
+          return this.isRegression_(this.modelType)
+            ? Math.abs(val1 - val2) < this.minCounterfactualValueDist
+            : val1 == val2;
+        },
+
+        adjustMaxCounterfactualValueDist_: function(selected, valueStr) {
+          this.maxCounterfactualValueDist = Math.max(
+            this.distanceStats_[valueStr].max -
+              this.visdata[selected][valueStr],
+            this.visdata[selected][valueStr] - this.distanceStats_[valueStr].min
+          );
+        },
+
+        adjustMinCounterfactualValueDist_: function() {
+          const valueStr = this.strWithModelName_(
+            inferenceValueStr,
+            this.nearestCounterfactualModelIndex
+          );
+          this.minCounterfactualValueDist = this.distanceStats_[
```

Comment:
Can this be a computed property of `modelType`, `distanceStats_` and `nearestCounterfactualModelIndex`? or does it need to only update when we are looking for the closest counter factual?

Reply:
`minCounterfactualValueDist` is actually controlled by the user. Here we're just defining its initial value.

Reply:
Ok, and I presume the initial value cannot be set in the property declaration?